platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/AdminConsoleService.java

```diff
@@ -570,9 +616,9 @@ public CardinalityEnforcer(Class<T> clazz) {
     }
 
     @SuppressWarnings("unchecked")
-    public T[] positiveCardinality(Object value) {
+    public Object[] positiveCardinality(Object value) {
```

Comment:
📘 Had to make this change because casting `String[]` to `T[]` was throwing runtime errors. This was the cleanest solution I could find.

Reply:
I'm curious about this, because as long as you expected back Object[] then it should have done the cast for you automatically.

Reply:
The return type on this method used to be `T[]`, and casting to a generic array is apparently rocket science 🤷‍♂

Reply:
Let me know if you want me to reproduce it yourself